SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialWorkerConnection.cpp

```diff
@@ -9,8 +9,87 @@
 #include "Async/Async.h"
 #include "Misc/Paths.h"
 
+#if WITH_EDITOR
+#include "Editor.h"
+#include "Settings/LevelEditorPlaySettings.h"
+#endif
+
 DEFINE_LOG_CATEGORY(LogSpatialWorkerConnection);
 
+#if WITH_EDITOR
+struct EditorWorkerController
+{
+	void OnPrePIEEnded(bool bValue)
+	{
+		LastPIEEndTime = FDateTime::Now().ToUnixTimestamp();
+		FEditorDelegates::PrePIEEnded.Remove(PIEEndHandle);
+	}
+
+	void InitWorkers(const FString& WorkerType)
+	{
+		ReplaceProcesses.Empty();
+
+		const int64 WorkerReplaceThreshold = 8;	// Issue replace worker request if restarting deployment within this timeframe
```

Comment:
magic number? 🤔 
Also can you put units here in the variable name

Reply:
Will add seconds to the name. Not a magic number per-say, will update the comment.